ExampleWallet/ExampleWallet/SeedPhraseValidator.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import MnemonicSwift
+
+protocol SeedPhraseValidator {
+    func isValid(seedPhrase: String) -> Bool
+}
+
+struct RealSeedPhraseValidator: SeedPhraseValidator {
+    func isValid(seedPhrase: String) -> Bool {
+        do {
+            try Mnemonic.validate(mnemonic: seedPhrase)
+            return true
+        } catch {
+            return false
+        }
+    }
+}
```

Comment:
Not I sure I see the utility of this. Why not just use `Mnemonic.validate` directly? That also seems to provide a useful error message.

Reply:
Two reasons: 1) To mimic the Android implementation, 2) To provide an abstraction layer, so that we can change the underlying implementation to another library later (BitcoinKit or whatever), without affecting the rest of the app.

Reply:
Yeah, it might be overkill to wrap the MnemonicSwift package if it turns out that this will be the only usage in the whole example app. 
However, in general I think it is a good practice to wrap third party libraries to get a looser coupling, and make it easier to switch the actual implementation / library. 
I also suspect that this interface will probably evolve slightly to wrap a few more Cryptographic methods (which may in fact be implemented by different libraries) - in that case it should probably also get a new name.